syntax="proto3";

import "Student.proto";

message StudentsGroup{
	repeated Student students = 1;
}